Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Indexer/NodeIndexer.php

```diff
@@ -367,7 +375,7 @@ public function removeNode(NodeInterface $node) {
 			)
 		);
 
-		$this->logger->log(sprintf('NodeIndexer: Removed node %s from index (node actually removed). Persistence ID: %s', $node->getContextPath(), $identifier), LOG_DEBUG, NULL, 'ElasticSearch (CR)');
+		$this->logger->log(sprintf('NodeIndexer: Removed %s from index (node actually removed). Persistence ID: %s', $node, $identifier), LOG_DEBUG, NULL, 'ElasticSearch (CR)');
```

Comment:
war im Original schon falsch, das letzte in dem String ist doch nicht der "Persistence ID" sondern der "Identifier". Das sind doch separate Sachen oder? Persistence ID ist doch die ID in der Datenbank, nach einem Reimport eine andere, "Identifier" ist das globale immer gültige was auch site export / import überlebt. Oder??